python/default_base_python_requirements.txt

```
@@ -1 +1,2 @@
 apache-beam[gcp]
```

If this is base for all future python templates, perhaps at least including dataframe is a good idea
```suggestion
apache-beam[gcp,dataframe]
```

---

I don't think dataframes are used by most python transforms, so I'd rather not add this unless we find we need it

---

Are they not used by most if not all FIleIO's? Just a suggestion, definitely not a blocker

---

No, just a couple - https://github.com/apache/beam/blob/707d204595e1b338daffacee2703fa46f038e858/sdks/python/apache_beam/io/textio.py#L1061